### ISSUE TITLE
Solving olympiad geometry without human demonstrations

### ISSUE THREAD

**Issue by 2nazero** [2nazero/T2GI-Survey#1]
## Title
**Solving olympiad geometry without human demonstrations**

## Links
- **[Paper Link](https://www.nature.com/articles/s41586-023-06747-5#Sec8)**
- **[Code Link](https://deepmind.google/discover/blog/alphageometry-an-olympiad-level-ai-system-for-geometry/)**
- [Blog](https://deepmind.google/discover/blog/alphageometry-an-olympiad-level-ai-system-for-geometry/)

## **Abstract**
Proving mathematical theorems at the olympiad level represents a notable milestone in human-level automated reasoning[1](https://www.nature.com/articles/s41586-023-06747-5#ref-CR1),[2](https://www.nature.com/articles/s41586-023-06747-5#ref-CR2),[3](https://www.nature.com/articles/s41586-023-06747-5#ref-CR3),[4](https://www.nature.com/articles/s41586-023-06747-5#ref-CR4), owing to their reputed difficulty among the world’s best talents in pre-university mathematics. Current machine-learning approaches, however, are not applicable to most mathematical domains owing to the high cost of translating human proofs into machine-verifiable format. The problem is even worse for geometry because of its unique translation challenges[1](https://www.nature.com/articles/s41586-023-06747-5#ref-CR1),[5](https://www.nature.com/articles/s41586-023-06747-5#ref-CR5), resulting in severe scarcity of training data. We propose AlphaGeometry, a theorem prover for Euclidean plane geometry that sidesteps the need for human demonstrations by synthesizing millions of theorems and proofs across different levels of complexity. AlphaGeometry is a neuro-symbolic system that uses a neural language model, trained from scratch on our large-scale synthetic data, to guide a symbolic deduction engine through infinite branching points in challenging problems. On a test set of 30 latest olympiad-level problems, AlphaGeometry solves 25, outperforming the previous best method that only solves ten problems and approaching the performance of an average International Mathematical Olympiad (IMO) gold medallist. Notably, AlphaGeometry produces human-readable proofs, solves all geometry problems in the IMO 2000 and 2015 under human expert evaluation and discovers a generalized version of a translated IMO theorem in 2004.

## BibTeX Citation
```bibtex
@article{trinh2024solving,
  title={Solving olympiad geometry without human demonstrations},
  author={Trinh, Trieu H and Wu, Yuhuai and Le, Quoc V and He, He and Luong, Thang},
  journal={Nature},
  volume={625},
  number={7995},
  pages={476--482},
  year={2024},
  publisher={Nature Publishing Group}
}
```
